x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/MetadataAttribute.java

```diff
@@ -28,6 +29,8 @@ public class MetadataAttribute extends TypedAttribute {
         tuple(DataTypes.KEYWORD, true),
         IdFieldMapper.NAME,
         tuple(DataTypes.KEYWORD, false), // actually searchable, but fielddata access on the _id field is disallowed by default
+        IgnoredFieldMapper.NAME,
+        tuple(DataTypes.KEYWORD, false),
```

Comment:
Should this be "searchable"? I'm not sure of the implications

Reply:
I'm unsure too! Maybe @astefan knows.

Reply:
> Should this be "searchable"? I'm not sure of the implications

If it's searchable, it'll be pushed down in a filter, if possible (`WHERE _ignored == "some field"`). However, this isn't always doable even if searchable and pushable (see `_id`), but it is in this case (and your tests cover that).